packages/block-library/src/gallery/edit.js

```diff
@@ -337,8 +340,10 @@ function GalleryEdit( props ) {
 		/>
 	);
 
+	const blockProps = useBlockProps();
+
 	if ( ! hasImages ) {
-		return mediaPlaceholder;
+		return <View { ...blockProps }>{ mediaPlaceholder }</View>;
```

Comment:
I've seen this pattern in a lot of component now. We should probably be able to pass down props to the media placeholder itself instead of wrapping it.